src/main/java/org/jetbrains/research/anticopypaster/utils/KeywordsMetrics.java

```diff
@@ -21,7 +21,7 @@ private float getKeywordsMetricFromFV(FeaturesVector fv){
         if(fv != null){
             float[] fvArray = fv.buildArray();
             int totalKeywords = 0;
-            for(int i = 16; i<=77; i+=2){
+            for(int i = 16; i<77; i+=2){
```

Comment:
I have seen this change a couple of times, why are we getting ride of the equals here?

Reply:
It's redundant.